tests/kubeaddons-enterprise/addons/generic-application-operator/0.x/generic-operator-install-0-x/00-assert.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: generic-application-operator
+
```

Comment:
Shouldn't we also assert something about the state?

Reply:
the addon installation of the generic operator itself doesn't create anything